src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -0,0 +1,800 @@
+{
+  "meta": {
+    "releasedOn": "2023.04.20"
+  },
+
+  "important": [
+    {
+      "what": [
+      "namespace import/export has some edge cases which prevent it from being exported.",
+      "Namely, one of them, is when you reference a deleted module from a page, that was crashes it."
+      ]
+    }
+  ],
+
+  "Added": [
+    {
+      "what": [
+        "the ability for users to disable the navigation sidebar within {APP_NAME_COMPOSE} namespace public pages.",
+        "The option is available under the `hide sidebar on namespace public pages` checkbox in the namespace edit screen."
+      ],
+      "why": [
+        "because we wanted to provide the option of disabling built-in navigation in favour of re-defining it with the new navigation page block."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/41ed9ed6ba8ab474cca7a512f04601ebf0ee4ecd"
+      ]
+    },
+    {
+      "what": [
+        "support for page duplication in edit mode and page builder view."
+      ],
+      "why": [
+        "because this speeds up the development process of the project and allows users to copy/share content from one page to another."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/pull/450/commits/9ea064c57552bb3c173302d6a4111e3889f92115",
+        "https://github.com/cortezaproject/corteza-locale/commit/47e06ebb251d87f3ff058e3a42dc7b26ef425ad5"
+      ]
+    },
+    {
+      "what": [
+        "auto-refreshing and manual refreshing feature to page blocks excluding, Content, Record and File Blocks.",
+        "An icon now appears on blocks when you enable the refresh feature.",
+        "That icon is clickable even though refresh will happen automatically based on the duration you have set."
+      ],
+      "why": [
+        "because users do not have to refresh the entire page to see new content on blocks and to give users more control over refreshing when auto-refreshing is not needed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-js/pull/73/commits/1a77d99e54697f1bbe29622688996dc05a8c06c4",
+        "https://github.com/cortezaproject/corteza/pull/760/commits/954419bac6c3d19e3c5c46adf93d908528ad18df"
+      ]
+    },
+    {
+      "what": [
+        "support for personalized auth screen and simplified smaller styling and background changes with the new admin webapp additions."
```

Comment:
I think this is too generic.
Instead of beating around the bush, can we just point out what was added and what it's for?

* The new admin webapp additions; ok what were they?